resources/compass/charts/postgresql/README.md

```diff
@@ -0,0 +1,204 @@
+# PostgreSQL
+
+## Overview
+
+[PostgreSQL](https://www.postgresql.org/) is an object-relational database management system (ORDBMS) with an emphasis on extensibility and on standards-compliance.
+
```

Comment:
Is this document copied 1:1 from any official Postgres documentation? If it is, maybe it would be a better idea just to provide a link?